src/app/services/age-verification/age-verification.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AgeVerificationService {
+  readonly LEGAL_AGE = 18;
+
+  ageVerification(age: number, firstName: string, lastname: string): string {
+    if (age < this.LEGAL_AGE) {
+      return `${firstName} ${lastname} is a minor`;
```

Comment:
Looks like you have repeatable code, you should always practice DRY(Dont repeat yourself). I will suggest you:

```typescript
const fullName = `${firstName} ${lastName}`;
const userHasLegalAge = age >= this.LEGAL_AGE;

if (userHasLegalAge) {
  return `${fullName} is legal age`;
}

return `${fullName} is a minor`;
```

And you can see I avoid `else` for less code ;)